app/src/main/java/org/mozilla/fenix/home/recenttabs/view/RecentTabs.kt

```diff
@@ -287,7 +294,7 @@ private fun RecentTabIcon(
                 alignment = alignment
             )
         }
-        else -> {
+        !inComposePreview -> {
```

Comment:
It would be nice if we can also add a default preview image for this RecentTabIcon. This would appear in bottom row with the URL secondary text.